### DISCUSSION TITLE
Enter transaction questions

### DISCUSSION THREAD

**Discussion by shinebrillant** [enrique-lozano/Monekin#253] [Ideas / New features]
After pressing the + sign, the default is expenditure. There needs to be a way to select income, expenditure or transfer after pressing the + sign.
When inputting values, a computer-like interface pops up. I put a word calculator for comparison. I feel that the inputted numbers are very small and confusing. I only need numbers, +-*/= and delete functions.

There is a problem with the sign of numerical input. You can directly display the sign next to the value and change it after calculation.

Negative numbers need to be allowed for transfer, just to facilitate the operation without having to select a secondary account:
Transfer +10
A to B
=A-10, B+10

Transfer-10
A to B
=B-10, A+10

"Amount transferred at destination" I don't know what function this is?

![ResizedImage_2024-11-13_21-36-14_9796](https://github.com/user-attachments/assets/138630fe-6bc3-4d76-9ef9-b5a965200883)

![Screenshot_20241113-220152](https://github.com/user-attachments/assets/f0314e53-59d6-4332-a67b-898755c3e72a)

**Comment by enrique-lozano** [comment]
I think that you are right and maybe we should make the calculator a little bigger. This is probably a great future setting, where you can configure the calculator height.

Negative transfers are a no-sense, and that's why in the calculator you can not put any negative value. The amount selected in the calculator is the amount transferred from the origin (which obviously can not be negative). The amount that arrive to the destination is the "Amount transferred at destination", which defaults to the same amount as the transferred from the origin. The label is little confusing and I'm planning to improve that soon.
